src/aks-preview/HISTORY.md

```diff
@@ -24,7 +24,6 @@ Release History
 +++++
 * Add --enable-azure-rbac and --disable-azure-rbac in aks update
 * Support disabling local accounts
-* Add addon `azure-defender` to list of available addons under `az aks enable-addons` command
```

Comment:
Since 0.5.12 has been released, maybe we should write about removing this addon in the new version such as 0.5.17?

Reply:
Well, since the addon wasn't integrated yet into AKS RP, then no one could have installed it anyways. Feels redundant to me

Reply:
If you want to make all your changes take effect, please upgrade version in setup.py. And here you need to add change in release note for new version and keep existing one.

Reply:
Resolved